### PR TITLE
Add video page type and directory

### PR DESCRIPTION
**Pull request by liamnewmarch**

### CODE REVIEW

File: developerportal/apps/people/templates/people.html [hunk 1]
```diff
@@ -20,5 +20,8 @@ <h2>People</h2>
         {% include "organisms/newsletter-signup.html" %}
       </section>
     </div>
+    <section id="newsletter" class="section-full section-background">
```
Comment (dcgauld) [inline developerportal/apps/people/templates/people.html]: Can you revert this change? Looks like a bad merge.

Comment (liamnewmarch) [reply developerportal/apps/people/templates/people.html]: Removed
